docs/guide/acquiring_data/helioviewer.rst

```diff
@@ -147,19 +147,15 @@ Map object.
 
 The overall syntax is similar to the *download_png* request, expect instead of
 specifying a single string to indicate which layers to use, here we
-can specify the values as separate keyword arguments.::
+can specify the source ID of the image we want.::
 
    >>> from sunpy.net.helioviewer import HelioviewerClient
-   >>> import matplotlib.pyplot as plt
-   >>> from astropy.units import Quantity
    >>> from sunpy.map import Map
    >>> hv = HelioviewerClient()  # doctest: +REMOTE_DATA
-   >>> filepath = hv.download_jp2('2012/07/05 00:30:00', observatory='SDO', instrument='HMI', detector='HMI', measurement='continuum')  # doctest: +REMOTE_DATA
+   >>> data_sources = hv.get_data_sources()  # doctest: +REMOTE_DATA
+   >>> filepath = hv.download_jp2('2012/07/05 00:30:00', sourceId=data_sources['SDO']['HMI']['continuum']['sourceId'])  # doctest: +REMOTE_DATA
    >>> hmi = Map(filepath)  # doctest: +REMOTE_DATA
-   >>> xrange = Quantity([200, 550], 'arcsec')  # doctest: +REMOTE_DATA
-   >>> yrange = Quantity([-400, 200], 'arcsec')  # doctest: +REMOTE_DATA
-   >>> hmi.submap(xrange, yrange).peek()  # doctest: +SKIP
```

Comment:
What was wrong with this?

Reply:
If we were using them then we were not able to show the image. It was giving a segmentation fault. And all we wanted to show in this example was that how can we download a jp2 image using the download_jp2 function so we don't really need that as it was just converting the image into a rectangle from circle